test/components/selectionBoxLayerTests.ts

```diff
@@ -0,0 +1,72 @@
+///<reference path="../testReference.ts" />
+
+var assert = chai.assert;
+
+describe("SelectionBoxLayer", () => {
+  it("bounds()", () => {
+    var svg = generateSVG();
+    var sbl = new Plottable.Component.SelectionBoxLayer();
+    sbl.renderTo(svg);
+
+    var topLeft: Plottable.Point = {
+      x: 100,
+      y: 100
+    };
+    var bottomRight: Plottable.Point = {
+      x: 300,
+      y: 300
+    };
+
+    function assertCorrectRendering(expectedTL: Plottable.Point, expectedBR: Plottable.Point, msg: string) {
+      var selectionBox = svg.select(".selection-box");
```

Comment:
Not sure it's that important, I mostly added it for legibility in the DOM inspector.